src/System.Management.Automation/engine/Attributes.cs

```diff
@@ -87,7 +87,7 @@ namespace System.Management.Automation
     /// <see cref="ValidateArgumentsAttribute"/> validates the argument as a whole. If the argument value may
     /// be an enumerable, you can derive from <see cref="ValidateEnumeratedArgumentsAttribute"/>
     /// which will take care of unrolling the enumerable and validate each element individually.
-    /// It is also recommended to override <see cref="System.Object.ToString"/> to return a readable string
+    /// It is also recommended to override <see cref="object.ToString"/> to return a readable string
```

Comment:
The `System.Object` full name looks better and matches what is in the documentation. I think it's right. Can we change only the code but not the XML comments?

Reply:
We would need `#pragma warning disable IDE0049 // Simplify Names` which would surely look worse. The [documentation](https://learn.microsoft.com/dotnet/api/system.management.automation.argumenttransformationattribute) appears to be built automatically from these comments.

Reply:
Generated docs look good.